src/App.tsx

```diff
@@ -1,7 +1,39 @@
-import React from 'react';
+import { Component } from 'react';
 
-export const App: React.FC = () => (
-  <div className="App">
-    <p className="App__message">The last pressed key is [Enter]</p>
-  </div>
-);
+interface State {
+  key: null | string,
+}
+
+export class App extends Component<{}, State> {
+  state: Readonly<State> = {
+    key: null,
+  };
+
+  componentDidMount(): void {
+    document.addEventListener('keyup', this.keyPressHandler);
+  }
+
+  componentWillUnmount(): void {
+    document.removeEventListener('keyup', this.keyPressHandler);
+  }
+
+  keyPressHandler = (event: KeyboardEvent) => {
+    const { key } = event;
+
+    this.setState({ key });
+  };
+
+  render() {
+    const { key } = this.state;
+
+    return (
+      <div className="App">
+        {
+          !key
+            ? <p className="App__message">Nothing was pressed yet</p>
+            : <p className="App__message">{`The last pressed key is [${key}]`}</p>
+        }
+      </div>
```

Comment:
```suggestion
      <div className="App">
        <p className="App__message">
          {
            key
              ? {`The last pressed key is [${key}]`}
              : Nothing was pressed yet
          }
        </p>
      </div>
```

Reply:
Done.
But I should wrap it on single quotes, BCS React thing I use it like a variable.